src/main/java/org/breedinginsight/brapps/importer/services/FileImportService.java

```diff
@@ -365,7 +365,7 @@ public ImportResponse updateUpload(UUID programId, UUID uploadId, AuthenticatedU
         importDAO.update(upload);
         // Redo the mapping
         //TODO: Get better errors for these
-        List<BrAPIImport> brAPIImportList;
+        List<BrAPIImport> brAPIImportList = null;
```

Comment:
Looks like this is set to `null` so that it can be checked before running the other processing later. Could you just put that chunk of code in this try statement to simplify the logic? You could always put lines 387-430 in another method to reduce code nesting and have it look cleaner. 

I'm thinking that this null check could potentially lead to bugs in the future if we add more stuff after this try/catch but forget to add a null check.